### DISCUSSION TITLE
*.componentName not specified

### DISCUSSION THREAD

**Discussion by hodossy** [visgl/deck.gl#7811] [Q&A]
After upgrading to 8.9.5, I started to get `deck: LayerExtension.componentName not specified` and `deck: PathStyleExtension.componentName not specified` logs from `create-props.js:194`. I have not found anything mentioned about component names in the docs. Is it something I can avoid. I looks awkward in production (where the class names are minified, making it even worse). How can I make these go away?

**Comment by Pessimistress** [comment]
Fixed in v8.9.7.
